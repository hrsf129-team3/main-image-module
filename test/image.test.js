import React from 'react';
import Image from '../client/src/components/image.jsx';


describe('Image', () => {
  it('should render image component', () => {
    const component = shallow(<Image image={[]}/>);
    expect(component).toMatchSnapshot();
  });
})

