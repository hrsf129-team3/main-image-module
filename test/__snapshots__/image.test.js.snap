// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render image component 1`] = `ShallowWrapper {}`;
